src/gluonts/torch/modules/loss.py

```diff
@@ -48,10 +49,31 @@ def forward(
 
 
 class NegativeLogLikelihood(DistributionLoss):
+    def __init__(self, beta: float = 0.0):
+        """
```

Comment:
Could you move this docstring to the class directly, instead of the constructor? So it’s consistent with the rest